.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor/
-composer.lock
\ No newline at end of file
+composer.lock
+tests/Fixtures
```

Comment:
also, ignoring the whole fixtures directory might not be the best option, are you trying to ignore the "var" directory that Symfony creates?

Reply:
> are you trying to ignore the "var" directory that Symfony creates?

there is `cache` and other stuff as well